src/scope/commandScope.ts

```diff
@@ -20,23 +21,29 @@ class ScopeImpl implements Scope {
   }
 }
 
-// TODO: Document the usage of these labels.
 const Command = 'command';
 const Job = 'job';
 
 export default class CommandScope extends ScopeIterator {
   *scopes(events: IterableIterator<Event>): Generator<Scope> {
+    let commandFound = false;
     for (const event of events) {
       if (
         event.isCall() &&
         (event.codeObject.labels.has(Command) ||
           event.codeObject.labels.has(Job) ||
           event.httpServerRequest)
       ) {
+        commandFound = true;
         yield new ScopeImpl(event);
 
         this.advanceToReturnEvent(event, events);
       }
     }
+    // If an AppMap contains no command events, then command scope should
+    // behave the same as root scope.
```

Comment:
Falling back on root scope seems better than emitting nothing at all from the scope.